app/models/form/lettings/pages/managing_organisation.rb

```diff
@@ -0,0 +1,32 @@
+class Form::Lettings::Pages::ManagingOrganisation < ::Form::Page
+  def initialize(id, hsh, subsection)
+    super
+    @id = "managing_organisation"
+    @header = ""
+    @description = ""
+    @subsection = subsection
+  end
+
+  def questions
+    @questions ||= [
+      Form::Lettings::Questions::ManagingOrganisation.new(nil, nil, self),
+    ]
+  end
+
+  # For an organisation owns and manages ONLY their own housing stock AND NOT uses agents to manage the properties AND NOT manages housing stock of other organisations?
+  # In "set up this lettings log" no extra question should be shown
```

Comment:
Similar to above, but this is different, right?

#No extra question should be shown in "set up this lettings log" if:
#the organisation owns and manages ONLY their own housing stock
#AND doesn't have managing agents